interchange/DeviceResources.capnp

```diff
@@ -339,6 +339,33 @@ struct Device {
         constant @2 : ConstantType;
     }
 
+    # These structures are used to define default constant values for unused
+    # or missing cell pins. For each cell type, we have a list of pins and
+    # what to do with those pins (which will be to tie it to 0 or 1 in most
+    # cases).
+    enum CellPinValue {
+      # leave floating
+      float        @0;
```

Comment:
I'm assuming we are intentionally being explicit here as I imagine (for Xilinx devices at least), the majority of pins will be `float`.  If we think spending the extra space is worth it, I can live with this decision.  Otherwise, we could just declare `float` as the implicit default.

Reply:
A small difference between `float` and no entry at all, is that `float` will create the cell pin if it's missing together but leave it floating which might matter in some cases and so I'd rather factor in.

In terms of space,  I am not so sure that the majority of pins will be `float`. Several of the large hard IPs like the PS7/8/9 have a default of ground for the vast majority of their pins (and vcc for most of the others, iirc); and they have thousands of pins.

Reply:
That make sense, Let's keep it as is.